elastic/kibana
# [Infrastructure UI] Hosts view charts fix

closes: [#160734](https://github.com/elastic/kibana/issues/160734)

## Summary

This PR fixes an inconsistency between the table and charts when there is no data to be displayed

<img width="600" alt="image" src="https://github.com/elastic/kibana/assets/2767137/a74429ab-a132-42eb-9bee-9d19fb2ad255">


The Lens charts were not receiving the `query` property from Unified Search, causing them to run a query with incomplete filters


### How to test

- Start a local Kibana instance
- Navigate to `Infrastructure > Hosts`
- Use the unified search using filters that won't match with any hosts
- Verify the charts
  -  Navigate from a KPI and Metrics charts to Lens
